lib/common/ipc_client.c

```diff
@@ -822,7 +822,7 @@ struct crm_ipc_s {
     unsigned int buf_size;     // size of allocated buffer
     int msg_size;
     int need_reply;
-    char *buffer;
+    GByteArray *buffer;
```

Comment:
As you noted in https://github.com/ClusterLabs/pacemaker/pull/3868#issuecomment-2855110325, my comments about `GByteArray` still apply to this commit.